app/services/food_db_service.rb

```diff
@@ -0,0 +1,19 @@
+class FoodDBService
+  def self.get_searched_foods(search_query)
+    response = conn.get("https://api.nal.usda.gov/fdc/v1/foods/search?api_key=#{ENV["food_api_key"]}&query=#{search_query}")
+    json = JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.get_top_hits(search_query)
+    response = conn.get("https://api.nal.usda.gov/fdc/v1/foods/search?api_key=#{ENV["food_api_key"]}&query=#{search_query}")
```

Comment:
Typically, the path here would only need to be `/fdc/v1/foods . . .` since the base URL is defined on line 15. This might not have been working with the connection however because the parameter on line 16 is named X-API-KEY instead of api_key